<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editForm</class>
 <widget class="QMainWindow" name="editForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameCarFormUpdate">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelUpdateCar">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>UpdateCar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelYear">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputYear">
         <property name="font">
          <font>
           <family>Ink Free</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMake">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Make</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputMake">
         <property name="font">
          <font>
           <family>Ink Free</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelModel">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputModel">
         <property name="font">
          <font>
           <family>Ink Free</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCountry">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputCountry">
         <property name="font">
          <font>
           <family>Ink Free</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPrice">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Price</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputPrice">
         <property name="font">
          <font>
           <family>Ink Free</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frameEditControls">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>125</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="updateBtn">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelUpdateBtn">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
